processor/src/services/adyen-payment.service.ts

```diff
@@ -155,6 +157,8 @@ export class AdyenPaymentService extends AbstractPaymentService {
             name: 'Adyen Apple Pay config check',
             status,
             ...(status === 'DOWN' && {
+              message:
+                'Apple Pay configuration is not complete, please fill in all the Apple Pay "own" environment variables',
```

Comment:
Since we have no `Error` object I copied over the same message as in the `details`.